go.mod

```diff
@@ -267,3 +267,5 @@ replace (
 	// as to why we are using a private fork.
 	sigs.k8s.io/controller-tools => github.com/cilium/controller-tools v0.8.0-1
 )
+
+replace github.com/sasha-s/go-deadlock => github.com/jrajahalme/go-deadlock v0.0.0-20231103061022-4ce5b1a047c2
```

Comment:
What is the reason for the new replace directive? We're trying very hard to get rid of all of them as they create a lot of issues for downstream projects.

Reply:
Intended to fix this before undrafting this, but forgot. We now have https://github.com/cilium/go-deadlock, will make this PR use that instead.

Reply:
Fixed.